src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+import javax.swing.*;
```

Comment:
The import statement for 'javax.swing.*' seems to be unused and should be removed for cleaner code.